runtime/filetype.vim

```diff
@@ -2209,6 +2209,9 @@ au BufNewFile,BufRead *.scala			setf scala
 " SBT - Scala Build Tool
 au BufNewFile,BufRead *.sbt			setf sbt
 
+" Slang
```

Comment:
ShaderSlang

Reply:
@chrisbra, the language is just 'slang' anyway, I'm not sure if its better to rename comments to reflect internal filetype.